lib/daru/dataframe.rb

```diff
@@ -1346,7 +1346,7 @@ def index= idx
     #   df.vectors = Daru::Index.new([:foo, :bar, :baz])
     #   df.vectors.to_a #=> [:foo, :bar, :baz]
     def vectors= idx
-      unless index.is_a?(Daru::Index)
+      unless idx.is_a?(Daru::Index)
```

Comment:
Please let me confirm:

The following code is **valid**:
```ruby
df.vectors = Daru::Index.new [:a, :b, :c]
```

And, the following code is **invalid** and should **throw error**:

```ruby
df.vectors = [:a, :b, :c]
```
Is this ok? Am I correct?

In addition, let me confirm one more thing.
The original code is `index.is_a?(Daru::Index)`,
and the code I wrote is `idx.is_a?(Daru::Index)`.